dom.go

```diff
@@ -268,8 +271,16 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 // function is not used directly but rather the elem subpackage (which is type
 // safe) is used instead.
 func Tag(tag string, m ...MarkupOrComponentOrHTML) *HTML {
+	return NamespacedTag(``, tag, m...)
```

Comment:
> can you explain the assertion that interpreted strings are simpler?

Good question, I'm happy to discuss it.

For me, when it comes to single-line strings, I generally always start with interpreted strings as a default, and "upgrade" to raw strings when doing so makes the string simpler. That's typically when the strings involve characters such as `"`.

So, by default, I would do `"foo"`, not <code>\`foo\`</code>.

For a short and simple string like that, both work equally well. The differences start to come out as soon as you add special characters. With interpreted strings, the special characters can/need to be escaped. But a raw string literal cannot contain the <code>`</code> character itself.

For multi-line strings, I consider both equally, and often go with raw strings, because they're more readable and look more like the string value they represent. If the raw string is very long and happens to contain a few <code>\`</code> characters, I can get around that by concatenating the raw string + <code>"\`"</code> + rest of raw string.

The absolute only potential _technical_ advantage I see in using interpreted string for empty string is the following. The `"` character is wider, and more vertical than <code>\`</code>. So with the two of them together, to me at least, I think it's more visible that `""` and `" "` are empty sting and single space. But with raw string, it's a tiny bit harder to tell whether <code>\`\`</code> is empty string or a single space.

However, in the end, I think the main deciding factor comes down to consistency. If you look at the Go source code, there are a total of 7753 hits for `""`:

![image](https://cloud.githubusercontent.com/assets/1924134/24519221/7aa0a06c-1552-11e7-9141-1faa256d2772.png)

But only 241 matches for <code>\`\`</code>, and many of those hits are false positives:

![image](https://cloud.githubusercontent.com/assets/1924134/24519339/c89cd196-1552-11e7-9fe8-8f15e0fdee21.png)

So, I prefer to go with the most popular representation because it's most what most people writing Go code tend to agree on. Given there are no strong technical reasons to prefer one over the other – they're pretty much equivalent – that's all I have to go on.